src/pages/LoginPage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export default function LoginPage() {
+  return (
+    <div className='reg-log-ctn'>
+      <h1>Login to your account</h1>
+      <form>
```

Comment:
There is a package called [React Hook Form](https://react-hook-form.com), it will help you with these forms